tools/run_tests/run_tests.py

```diff
@@ -490,6 +490,14 @@ def makefile_name(self):
             return 'Makefile'
 
     def _clang_make_options(self, version_suffix=''):
+        if self.args.config == 'ubsan':
```

Comment:
CC @nicolasnoble 

I'm actually not sure why UBSAN is a special case. If anything, I expect all of the sanitizers to break. Before this change, UBSAN wouldn't build from a linking error because it would use `clang` instead of `clang++` as the `LD`. Our Makefile specifies that the `LD` for all sanitizers should be `clang++`, but I'm guessing `run_tests.py` overrides that behavior.